### PR TITLE
Developers Session v0.2

### PR DESCRIPTION
**Pull request by dz0ny**
ref: https://github.com/niteoweb/operations/issues/826

### CODE REVIEW

File: 2_Operations/work-process.md [hunk 1]
```diff
@@ -62,14 +62,17 @@ As an attendee, ask yourself:
...
 - Have I seen a great technical talk and I want to share it so we can discuss it the next time?
+- Do I want to share something I leared in a short lighning talk?
+- Do I need help with the user story I have been asigned to?
 
 As a curator:
 
-- Announce in #operations Zoom session to join into
+- Announce in #development Zoom session to join into
 - As a curator, meet and greet everyone, especially first-time joiners. 
+- Annonce who is having Lightning talk if one was prepared.
```
Comment (dmurko) [inline 2_Operations/work-process.md]: Announce

Comment (dmurko) [reply 2_Operations/work-process.md]: the Lightning talk

File: 2_Operations/work-process.md [hunk 2]
```diff
@@ -62,14 +62,17 @@ As an attendee, ask yourself:
...
 - If you hear crickets, start rolling the ball with a question, an anecdote or a funny story.
 - Make sure no one is talking the whole 30 minutes, everyone should have an opinion!
-- After 30 minutes or less announce that there is time for Lightning talks.
+- Write a report into Recurring ticket for the meeting.
```
Comment (dmurko) [inline 2_Operations/work-process.md]: I'd write "into _the_ recurring ticket" even if Grammarly doesn't raise it here.

File: 2_Operations/work-process.md (hunk 1 above)
Comment (dz0ny) [inline 2_Operations/work-process.md]:
```suggestion
- Announce who is having the Lightning talk if one was prepared.
```

File: 2_Operations/work-process.md (hunk 2 above)
Comment (dz0ny) [inline 2_Operations/work-process.md]:
```suggestion
- Write a report into the **Recurring** ticket for the meeting.
```

**Comment by dz0ny** [comment]
> missing day and time

They are in recurring task

**Comment by dmurko** [comment]
They're also in this doc. :)

Screenshot 2019-03-10 at 18 59 29

**Pull request merged**